tester/consumergroup.go

```diff
@@ -237,15 +237,15 @@ func (cgs *cgSession) MarkOffset(topic string, partition int32, offset int64, me
 	cgs.mMessages.Lock()
 	defer cgs.mMessages.Unlock()
 
-	msgKey := cgs.makeMsgKey(topic, offset)
+	msgKey := cgs.makeMsgKey(topic, offset-1)
```

Comment:
Maybe add a panic on `offset < 1`

Reply:
but this is totally tester internal specific? the offset is generated by the tester and it's also only used by the tester.